_data/other.yml

```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://www.ulule.com/
+      twitter: ulule
+      facebook: ulule
```

Comment:
I'm not saying it redirects to another facebook page. Just that `ulule` seems to be a French version.

ulule:
![ulule-screenshot](https://cloud.githubusercontent.com/assets/3535780/25050147/149393ba-2146-11e7-83f8-1a618cf14c6f.png)
ululeEn:
![ululeEn-screenshot](https://cloud.githubusercontent.com/assets/3535780/25050157/21da4ae6-2146-11e7-88ad-1c961d1ba80f.png)